src/PrestaShopBundle/Resources/config/services/form/form_data_provider.yml

```diff
@@ -0,0 +1,8 @@
+services:
+    _defaults:
+        public: true
+
+    prestashop.admin.customer_preferences.data_provider:
```

Comment:
I'm not against introduction of new configuration files (this is the way to go!) but if you do it, make a commit that move all the current definitions. This way we wont need to find form handlers/data providers everywhere.

Reply:
done.